src/core/backbone_router/bbr_manager.cpp

```diff
@@ -33,6 +33,9 @@
 
 #include "bbr_manager.hpp"
 
+#include <algorithm>
```

Comment:
It's for std::min, and std::max.

Reply:
Defined `OT_MIN` and `OT_MAX` in common/code_utils.hpp`.

Using <algorithm> had been fine. The only issue that could happen is conflict of `operator new` since it's both defined in STL and `common/new.hpp`. Do you think simple utilities like `std::min`, `std::max` should be allowed ?